intro/Readme.md

```diff
@@ -1,12 +1,12 @@
 # Intro Chapter
 
-Questions:
+## Questions:
 
 >What are the three most essential skills that a beginner programmer needs to know?
 
-### Go through each one of the exercises, 
-### type in each sample exactly as it is shown,
-### make it run/execute without errors.
+### Have the skill of reading and writing, 
+### attention for the details,
+### and noticing when things change.
```

Comment:
You corrected the three most essential skills, but you didn't change the explanation of each skill.

Please explain:

`reading and writing`  Why is this an important skill?

`attention to details`  Why do developers need to have this skill?  What happens if they don't?

`noticing differences`  Why?